sources/packages/backend/apps/queue-consumers/src/processors/schedulers/cas-integration/_tests_/cas-supplier-integration.scheduler.e2e-spec.ts

```diff
@@ -276,7 +267,7 @@ describe(describeProcessorRootTest(QueueNames.CASSupplierIntegration), () => {
     });
   });
 
-  it(
+  it.only(
```

Comment:
Is there a reason to keep this it.only ?

Reply:
Yeah, lack of attention 😉

Reply:
Removed.